KomaMRICore/test/runtests.jl

```diff
@@ -1,9 +1,15 @@
 using TestItems, TestItemRunner
 
+#JLD2, MAT, MRIFiles, Pkg, Reexport, Scanf
+
+
 @run_package_tests filter=ti->!(:skipci in ti.tags)&&(:core in ti.tags) #verbose=true
 
 @testitem "Sequence" tags=[:core] begin
     using Suppressor
+    include("../../KomaMRIIO/src/KomaMRIIO.jl")
+    using .KomaMRIIO
```

Comment:
Do this properly

Reply:
Addressed in [a7145d6](https://github.com/cncastillo/KomaMRI.jl/commit/a7145d642a633d7aa5d238007b66511b91241eda)